### PR TITLE
Feat single product page

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Created single product page for all the products
- Add to cart and Add to Wishlist functionality working.

### CODE REVIEW

File: src/pages/ProductPage/ProductFilters.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ProductFilters = () => {
     setShowFilters(!showFilters);
   };
   return (
-    <aside className="filters-sec p-2" style={{ height: "85vh" }}>
+    <aside className="filters-sec p-2">
```
Comment (saadahmadhere) [inline src/pages/ProductPage/ProductFilters.jsx]: good that you removed inline styling when it was possible

File: src/pages/single product page/SingleProductPage.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    const getSingleProduct = (products, productId) => {
+        return products.find((product) => product._id === productId)
+    }
+    
+    const product = getSingleProduct(products, productId)
+
+    return (
+        <>
+        <Navbar />
+        <SingleProductCard {...product} />
```
Comment (saadahmadhere) [inline src/pages/single product page/SingleProductPage.jsx:19]: passing props like this, was new to me.

**Review by saadahmadhere — COMMENTED** [review]
Beautifully written code, learned a few new thing reviewing you code.

**Review by MaruthiPothuganti — COMMENTED** [review]
Learned how to implement a Single Product Page. Good implementation.

**Pull request merged**